apps/core/src/utils/calculateAPY.ts

```diff
@@ -22,18 +22,17 @@ export function calculateAPY(
     if (stakingPoolActivationEpoch || stakingPoolActivationEpoch === 0) {
         const numEpochsParticipated = epoch - stakingPoolActivationEpoch;
         apy =
-            Math.pow(
+            (Math.pow(
                 1 +
                     (+stakingPoolSuiBalance - +poolTokenBalance) /
                         +poolTokenBalance,
-                365 / numEpochsParticipated
-            ) - 1;
+                        (365 / numEpochsParticipated)
+            ) - 1) * 100;
```

Comment:
Can you provide context about this change in the PR description?

Reply:
Posting it on Slack